nox.py

```diff
@@ -46,6 +47,7 @@ def _install_dev_packages(session):
 
 
 def _install_test_dependencies(session):
+    session.install('gevent')
```

Comment:
This is not needed. `_install_test_dependencies` is intended for test harness and utils.
`gevent` will be installed by the line you added - `session.install('-e', 'contrib/opencensus-ext-gevent')`.

Reply:
Ah, right. Thanks.